modules/delegates/api/fetchChainDelegates.ts

```diff
@@ -12,28 +12,44 @@ import { DelegateContractInformation } from '../types';
 import { gqlRequest } from 'modules/gql/gqlRequest';
 import { allDelegates } from 'modules/gql/queries/subgraph/allDelegates';
 import { networkNameToChainId } from 'modules/web3/helpers/chain';
+import { getPublicClient } from 'modules/web3/helpers/getPublicClient';
+import { chiefAbi, chiefAddress } from 'modules/contracts/generated';
 
 export async function fetchChainDelegates(
   network: SupportedNetworks
 ): Promise<DelegateContractInformation[]> {
   const chainId = networkNameToChainId(network);
   const data = await gqlRequest({
     chainId,
-    useSubgraph: true,
     query: allDelegates
   });
 
-  return data.delegates.map(d => {
-    // Ensure blockTimestamp is a valid number
-    const blockTimestamp = d.blockTimestamp ? Number(d.blockTimestamp) : 0;
-
-    return {
-      blockTimestamp,
-      address: d.ownerAddress,
-      voteDelegateAddress: d.id,
-      mkrDelegated: formatValue(BigInt(d.totalDelegated), 'wad', 18, false),
-      delegateVersion: Number(d.version),
-      lastVoteDate: d.voter?.lastVotedTimestamp ? Number(d.voter.lastVotedTimestamp) : null
-    };
```

Comment:
@tyler17 @hernandoagf you guys worked on this file recently, I tried to fix the merge conflict just want to get another pair of eyes to make sure I didn't miss anything

Reply:
Hm, the removal of `useSubgraph: true` seems wrong? When I load the deployment for this PR, it will sometimes crash on me, and the percentage of MKR delegated stat in the sidebar on the delegates page doesn't load for me

Reply:
good catch, my typescript parser incorrectly reported that `useSubgraph` wasn't a valid parameter